index.html

```diff
@@ -76,136 +76,160 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see you!</h1>
                     <li>
                         <img src="images/Counter.png"/>
                         Back End Dev
                     </li>
                     <li>
                         <img src="images/Counter.png"/>
                         2022
-                    </li>
+                    </li> -->
                 </ul>
+                </div>
                 <div class="second-para">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p class="project-description"></p>
+                    <!-- <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p> -->
                 </div>
-                <ul>
-                    <li class="lists">html</li>
+                <div class="web-technologies">
+                <ul class="lists">
+                    <!-- <li class="lists">html</li>
                     <li class="lists">css</li>
-                    <li class="lists">javascript</li>
-                </ul><br/><br/>
+                    <li class="lists">javascript</li> -->
+                </ul>
+                </div><br/><br/>
                 <div>
-                    <a href=""><button id="my-btn">See Project</button></a>
+                    <button class="project-btn" id="my-btn">See Project</button>
                 </div>
             </div>
         </section><br/><br>
         <section class="fourth-section">
-            <div>
-                <img src="images/Snapshoot Portfolio3.png" class="section-img"/>
+            <div class="section-img card-top">
+                <!-- <img src="images/Snapshoot Portfolio3.png" class="section-img"/> -->
             </div>
             <div class="second-section-div">
                 <div>
-                    <h2 class="my-h2">Tonic</h2>
+                    <h2 class="my-h2 project-title"></h2>
+                    <!-- <h2 class="my-h2">Tonic</h2> -->
                 </div>
+                <div class="card-categories">
                 <ul class="section-list">
-                    <li class="other-list" id="li-1">CANOPY</li>
+                    <!-- <li class="other-list" id="li-1">CANOPY</li>
                     <li>
                         <img src="images/Counter.png"/>
                         Back End Dev
                     </li>
                     <li>
                         <img src="images/Counter.png"/>
                         2022
-                    </li>
+                    </li> -->
                 </ul>
+                </div>
                 <div class="second-para">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p class="project-description"></p>
+                    <!-- <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p> -->
```

Comment:
- [ ] Please remove this and other commented unused code for readability purpose.